PROTOCOL.md

```diff
@@ -424,6 +424,37 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server.
+
+`|/team NUMBER|REQUEST ID`
+
+> Chooses which Pokemon to lead with.
+
+`|/choose move NUMBER|REQUEST ID` or `|/choose switch NUMBER|REQUEST ID`
+
+> Requests to use the selected move or switch to the selected Pokemon. The `REQUEST ID`
+> is used to identify which action the request was intended for to protect from cases
+> of trying to send multiple requests for the same action, which would otherwise effect
+> future decisions. The first decision of a match has a `REQUEST ID` of 1, and each
+> successful decision increments the counter by 1.
+> When a Pokemon is ko'd `/choose switch` is used to determine their replacement.
+
+`|/choose ACTION1 TARGET1,ACTION2 TARGET2|REQUEST ID`
+> For double and triple battles, decisions are sent for all team positions in the same
+> line separated by commas. Moves which specify a target list their chosen target as a
+> number wherein the opposing Pokemon are positive integers starting from `1` on the left
+> and incrementing up by one as they move to the right. Ally Pokemon targets are negative
+> integers starting from `-1` on the left and decrementing by one to the right. Moves
+> which don't need to specify a target send only the action.
+> If not all positions are making a decision, `pass` is sent in place of the action for
+> the position to be skipped.
```

Comment:
This and the above entry could possibly be merged, especially since you don't mention what `ACTION` is supposed to be in this entry and that needs clarification. If you do merge, say that just like how multi-target/self-target moves don't need to mention a target, you don't need a target in Singles, either.

Either way, I also remember there was some sort of Triples-specific command for swapping the positions of Pokemon that are already on the field but the `ACTION` involved escapes me at the moment. You should probably look into that.

You should mention that the move/switch numbers start from 1.

Also, a few Singles-only shorthand commands: `/move NUMBER` is shorthand for `/choose move NUMBER` and the same for `/switch NUMBER`.